.gitlab-ci.yml

```diff
@@ -65,15 +65,15 @@ cwl_dependency_is_stand_alone:
   stage: linting_and_dependencies
   script:
     - pwd
-    - virtualenv -p ${MAIN_PYTHON_PKG} venv && . venv/bin/activate && make prepare && make develop extras=[cwl]
+    - virtualenv -p ${MAIN_PYTHON_PKG} venv && . venv/bin/activate && pip install pip --upgrade pip==21.3.1 && make prepare && make develop extras=[cwl]
     - make test tests=src/toil/test/docs/scriptsTest.py::ToilDocumentationTest::testCwlexample
 
 
 wdl_dependency_is_stand_alone:
   stage: linting_and_dependencies
   script:
     - pwd
-    - virtualenv -p ${MAIN_PYTHON_PKG} venv && . venv/bin/activate && make prepare && make develop extras=[wdl]
+    - virtualenv -p ${MAIN_PYTHON_PKG} venv && . venv/bin/activate && pip install pip --upgrade pip==21.3.1 && make prepare && make develop extras=[wdl]
```

Comment:
It looks like `pip` is outdated enough that you can't use pip to upgrade pip in python3.10 so we're forced to use the bootstrap method.  I think we're probably safe to leave this unpinned and just write in: `curl https://bootstrap.pypa.io/get-pip.py | python3.10`, so the full line change here should be something like:

```
virtualenv -p ${MAIN_PYTHON_PKG} venv && . venv/bin/activate && curl https://bootstrap.pypa.io/get-pip.py | ${MAIN_PYTHON_PKG} && make prepare && make develop extras=[wdl]
```